docs/pages/router/advanced/custom-tabs.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: Custom Tab layouts
```

Comment:
```suggestion
title: Custom tab layouts
```
We usually only capitalize first letter, unless it's a product name / proper noun, right?

Reply:
You are absolutely right. My bad. I'll get this fixed.